ASR: typo fix from #724 (#727)
huggingface/huggingface.js: packages/tasks/src/tasks/automatic-speech-recognition/about.md

data = query("sample1.flac")
```

> You can also use [huggingface.js](https://github.com/huggingface/huggingface.js), the JavaScript client, to transcribe audio with the Serverless Inference API.

Reviewer: Oh damnn, I'm blind! thanks! 🤗

});
```

> This works for transformers-compatible models like Whisper, Wav2Vec2, HuBERT, etc. You can also run inference in Python using transformers as follows:

Reviewer: This reads a bit off to me :/

Reviewer: Resolved in the latest commit!

> This works for transformers-compatible models like Whisper, Wav2Vec2, HuBERT, etc. You can also run inference in Python using transformers as follows:

Reviewer 2: Mmm both are unclear to me, maybe this?

```suggestion
For transformers-compatible models like Whisper, Wav2Vec2, and HuBERT, you can also run inference with the library as follows:
```

Author: haha, I thought the original intent was to say that the previous api methods work for transformers models. But it looks fine like this imo.

Author: ping @Vaibhavs10 for confirmation
